react/wbooks/src/services/AuthServices.js

```diff
@@ -0,0 +1,12 @@
+import { signUpApi } from '../config/api';
+import { normalizeCreateUser } from '../app/utils/AuthUtils';
+
+export const serviceCreateUser = body => signUpApi.post('/users', body);
+
+export const createUser = async user => {
+  const userNormalize = normalizeCreateUser(user);
+  const userCreated = await serviceCreateUser(userNormalize);
+  return userCreated.ok ? { ok: true, data: userCreated.data } : { ok: false, problem: userCreated.problem };
+};
```

Comment:
Just for you to know. Here you can use then instead of await.
```
export const createUser = user => 
  serviceCreateUser(normalizeCreateUser(user)).then(({ ok, data, problem }) => ok ? { ok: true, data } : { ok: false, problem };
```
We try using await to avoid having many then nested one inside another. But with only 1 we can use then perfectly